charts/kubewarden-controller/templates/rbac.yaml

```diff
@@ -96,9 +99,15 @@ rules:
   - "apps"
   resources:
   - deployments
+  - replicasets
   verbs:
+  - get
   - list
   - watch
+  - create
+  - delete
+  - patch
+  - update
```

Comment:
We can get rid of this ones.

Reply:
Yes, I pushed the wrong file version. I've pushed a file that I used during some investigation about the permissions issues.